### DISCUSSION TITLE
gentoo 升级后dwl里鼠标指针进入 fcitx5 选词框 segfault

### DISCUSSION THREAD

**Discussion by guyuming76** [fcitx/fcitx5#924] [Q&A]
我最近升级了 gentoo 系统,  看到 qtwayland 等有小的升级, fcitx5貌似从gentoo-zh库变到了gentoo官方库, wlroots也有小的升级.

dwl上游的提交最近几个月我都没有合并到我的中文输入分支里.

下面链接我保存了升级后鼠标进入选词框后segfault的callstack截图,(貌似DWL为那个选词框 定义的border为空) ,以及部分猜测相关软件包升级版本截图:
https://gitee.com/guyuming76/dwl-dev-pic-and-video/commit/308c041c8656cb69bc6551cb0c936a4cc7a41746

**Comment by wengxt** [comment]
谁 crash 就是谁的问题。既然是 dwl crash 那就是它的问题。

**Reply by guyuming76** [reply]
我可能问的方式不对,直接抛了个crash callstack, 我也就是突然面对大量要排查的不熟悉领域,来找专家寻求帮助的.

下面链接里面我贴了些 升级后 fcitx5 安装的gentoo use flag配置截图,陈述了下自己的分析, 我不知道新版本有没有使用 gtk 画输入法候选框的功能,以便我排除qt升级的影响. 那些个qt5 的use flag都是干啥的?

因为一不小心dwl就crash,之前打了半天字就丢了,只能在本地一边打,一边存, commit 到git后再链接贴出来.

[https://gitee.com/guyuming76/dwl-dev-pic-and-video/commit/4f05268e11f133aaf63e102e9899391605f2bb75](https://gitee.com/guyuming76/dwl-dev-pic-and-video/commit/4f05268e11f133aaf63e102e9899391605f2bb75)
